pkg/generic/httpthrift_codec.go

```diff
@@ -46,15 +47,30 @@ type HTTPRequest = descriptor.HTTPRequest
 type HTTPResponse = descriptor.HTTPResponse
 
 type httpThriftCodec struct {
-	svcDsc           atomic.Value // *idl
-	provider         DescriptorProvider
-	codec            remote.PayloadCodec
-	binaryWithBase64 bool
+	svcDsc                          atomic.Value // *idl
+	provider                        DescriptorProvider
+	codec                           remote.PayloadCodec
+	binaryWithBase64                bool
+	dynamicgoConvOpts               conv.Options   // conversion options for dynamicgo
+	dynamicgoConvOptsWithThriftBase conv.Options   // conversion options for dynamicgo with EnableThriftBase turned on
```

Comment:
There is no need to define multiple options repeatedly here, the repeated definition is binaryConv, binaryConvWithThriftBase.........

Reply:
got it

Reply:
fixed